components/FormCountryPage/index.js

```diff
@@ -0,0 +1,69 @@
+import styled from "styled-components";
+import AppName from "../AppName";
+
+export default function FormCountry({ submitNewCountry }) {
+  function onSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const country = Object.fromEntries(formData);
+
```

Comment:
You could probably add an id and create your new country object here, rather than at the top level `app.js` inside the `submitNewCountry` function. That function would then only set the state variable and push to the designated page.